### PR TITLE
stop adding beta admission config to default master configs

### PR DESCRIPTION
**Pull request by deads2k**
Stop adding the image admission config.  I confirmed with @sdodson last week that he doesn't use the config this creates.

@liggitt as discussed last week.
@juanvallejo this snips the pkg/oc link you asked about before.

### CODE REVIEW

File: pkg/cmd/server/start/master_args.go [hunk 1]
```diff
@@ -362,23 +359,6 @@ func (args MasterArgs) BuildSerializeableMasterConfig() (*configapi.MasterConfig
 		config.ServiceAccountConfig.PublicKeyFiles = []string{}
 	}
 
-	// embed a default policy for generated config
```
Comment (liggitt) [inline pkg/cmd/server/start/master_args.go]: no objections from me. @smarterclayton, are we expecting this admission plugin to be enabled by default, and if so, does the default config for this plugin do what we want?

Comment (deads2k) [reply pkg/cmd/server/start/master_args.go]: > no objections from me. @smarterclayton, are we expecting this admission plugin to be enabled by default, and if so, does the default config for this plugin do what we want?

It does not.  No config means nothing.  This changes what an empty `openshift start master` does, but no one should be using that in production and production builds its config differently anyway.

**Comment by deads2k** [comment]
@liggitt integration I'll fix, test-cmd I'll remove.  comments?

**Comment by mfojtik** [comment]
@deads2k the cmd error seems real.. did this broke the image resolution?

**Comment by juanvallejo** [comment]
Closing in favor of https://github.com/openshift/origin/pull/17516 per the discussion with David in https://github.com/openshift/origin/pull/17486#discussion_r153822615

**Pull request closed without merging**